WooCommerce/Classes/ViewRelated/Products/Edit Product/Reviews/ProductReviewsDataSource.swift

```diff
@@ -39,10 +39,31 @@ final class ProductReviewsDataSource: NSObject, ReviewsDataSource {
         return reviewsResultsController.isEmpty
     }
 
+    /// Notifications
+    ///
+    private lazy var notificationsResultsController: ResultsController<StorageNote> = {
```

Comment:
I know that in some places we call them `notifications` because they are considered `notes`, but I believe that it's just an old term that its super confusing since they are just product reviews. What do you think renaming all these new references to `reviews`?

Reply:
These are actually the WordPress.com notifications linked to the reviews. We need them so we can mark them as read in the review detail view. It is true that it was a bit confusing because there wasn't a lot of documentation, I enhanced it with more information 🕺